doc/topics/windows/windows-package-manager.rst

```diff
@@ -1317,280 +1265,63 @@ following:
         - password: CorrectHorseBatteryStaple
 
 .. note::
-    Per-remote configuration settings work in the same fashion as they do in
+    The per-remote configuration settings work in the same manner as they do in
     gitfs, with global parameters being overridden by their per-remote
-    counterparts. For instance, setting :conf_master:`winrepo_passphrase` would
-    set a global passphrase for winrepo that would apply to all SSH-based
+    counterparts. For instance, setting :conf_master:`winrepo_passphrase`
+    sets a global passphrase for winrepo that applies to all SSH-based
     remotes, unless overridden by a ``passphrase`` per-remote parameter.
 
-    See :ref:`here <gitfs-per-remote-config>` for more a more in-depth
+    See :ref:`here <gitfs-per-remote-config>` for detailed
```

Comment:
**Nitpick:** Add "a" in front of "detailed."

Reply:
Updated